app/helpers/react_components/editor.rb

```diff
@@ -10,7 +10,7 @@ def to_s
           track_title: solution.track.title,
           exercise_title: solution.exercise.title,
           introduction: introduction,
-          instructions: SerializeExerciseInstructions.(solution.exercise),
+          assignment: SerializeExerciseAssignment.(solution.exercise),
```

Comment:
@kntsoriano Worth you seeing this specific commit in this PR. The word `instructions` ended up being too overloaded, as its a specific file to maintainers, but here it also has tests and other things. So we renamed it here to `assignment`.